dataland-batch-manager/src/test/kotlin/org/dataland/batchmanager/service/GleifGoldenCopyIngestorTest.kt

```diff
@@ -113,8 +117,11 @@ class GleifGoldenCopyIngestorTest {
         `when`(File.createTempFile(anyString(), anyString())).thenReturn(mock(File::class.java))
         val mockFileUtils = mockStatic(FileUtils::class.java)
         companyIngestor.processFullGoldenCopyFileIfEnabled()
-        mockStaticFile.verify({ File.createTempFile(any(), any()) }, times(2))
+        // One file is created for LEI update, one file for ISIN mapping and one for the relationship update
+        val numberOfDownloadedFiles = 3
```

Comment:
remove comment and consider to put variable on the top as a constant

Reply:
as it is only relevant for that particular test, I would prefer to keep the val definition here. What do you think?

Reply:
i understand your point, but you could move it to the top of this particular test at least.
nevertheless, it doesn't need to be on top of the whole file.

Reply:
nevermind, you can leave it here, too. Fine by me